### PR TITLE
add correct content type for pdf upload in exam_templates_controller_spec.rb

### PR DESCRIPTION
**Pull request by hemmatio**
## Proposed Changes
Currently, the method create_with_file() of exam_template.rb under /app/models is not covered by the rspec tests. The coverage can be seen [here](https://coveralls.io/builds/69616320/source?filename=app%2Fmodels%2Fexam_template.rb#L24). This was fixed by adding the correct content type of 'application/pdf' to the rspec test which attempts to create an exam template.

<details>
<summary>Screenshots of your changes (if applicable)</summary>
image
</details>

<details>
<summary>Associated <a href="https://github.com/MarkUsProject/Wiki">documentation repository</a> pull request (if applicable)</summary>
N/A
</details>

## Type of Change
*(Write an `X` or a brief description next to the type or types that best describe your changes.)*

| Type                                                                                    | Applies? |
|-----------------------------------------------------------------------------------------|----------|
| 🚨 *Breaking change* (fix or feature that would cause existing functionality to change) |          |
| ✨ *New feature* (non-breaking change that adds functionality)                          |          |
| 🐛 *Bug fix* (non-breaking change that fixes an issue)                                  |          X|
| 🎨 *User interface change* (change to user interface; provide screenshots)              |          |
| ♻️ *Refactoring* (internal change to codebase, without changing functionality)          |          |
| 🚦 *Test update* (change that *only* adds or modifies tests)                            |         X|
| 📦 *Dependency update* (change that updates a dependency)                               |          |
| 🔧 *Internal* (change that *only* affects developers or continuous integration)         |          |

## Checklist
*(Complete each of the following items for your pull request. Indicate that you have completed an item by changing the `[ ]` into a `[x]` in the raw text, or by clicking on the checkbox in the rendered description on GitHub.)*

Before opening your pull request:

- [X] I have performed a self-review of my changes.
  - Check that all changed files included in this pull request are intentional changes.
  - Check that all changes are relevant to the purpose of this pull request, as described above.
- [ ] I have added tests for my changes, if applicable.
  - This is **required** for all bug fixes and new features.
- [X] I have updated the project documentation, if applicable. (N/A in this case)
  - This is **required** for new features.
- [ ] If this is my first contribution, I have added myself to the list of contributors.

After opening your pull request:

- [ ] I have updated the project Changelog (this is required for all changes).
- [ ] I have verified that the pre-commit.ci checks have passed.
- [ ] I have verified that the CI tests have passed.
- [ ] I have reviewed the test coverage changes reported by Coveralls.
- [ ] I have [requested a review](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/requesting-a-pull-request-review) from a project maintainer.

## Questions and Comments
- There currently do not seem to be any tests for creating an exam template with a nil file or a file with a non-pdf content type. Perhaps these test cases can be created.
- Note to myself: How to create test cases for a change to a test case?
- I will add myself to the list of contributors before opening this pull request.

### CODE REVIEW

**Comment by david-yz-liu** [comment]
Hi @hemmatio, in response to your presentation yesterday: you are definitely on the right track, and please *add some additional test cases* to cover the edge cases you've described in your PR description comments, and possibly any others you come up with. 😄

**Comment by hemmatio** [comment]
image

I should add another test case to expect this error message.

File: Changelog.md [hunk 1]
```diff
@@ -10,6 +10,8 @@
 
 ### 🐛 Bug fixes
 
+- Fix test coverage for ExamTemplate.create_with_file method (#7213)
```
Comment (david-yz-liu) [inline Changelog.md:13]: Improving test coverage is considered an internal change, not a bug fix

**Review by david-yz-liu — APPROVED** [review]

**Comment by david-yz-liu** [comment]
Thank you, @hemmatio!
